aggregator/aggregator.go

```diff
@@ -124,7 +123,7 @@ type aggregatorMetrics struct {
 	gauges                    tally.Counter
 	invalidMetricTypes        tally.Counter
 	addMetricWithPoliciesList instrument.MethodMetrics
-	shards                    aggregatorShardMetrics
+	shards                    aggregatorShardsMetrics
```

Comment:
rename shards to shardMetrics? or metricPerShard?

Reply:
Can rename to `shardsMetrics` sure thing.

Reply:
Actually it's already inside an `aggregatorShardsMetrics` struct so when referenced it 'll look like `agg.metrics.shards.add.Inc(1)`, which seemed fairly self-explanatory and `agg.metrics.shardsMetrics.add.Inc(1)` seems a bit too verbose. I'll keep it as is for now and let me know if you think otherwise.

Reply:
Sounds good